WooCommerce/Classes/ViewRelated/Products/Filters/FilterProductListViewModel.swift

```diff
@@ -52,6 +52,8 @@ final class FilterProductListViewModel: FilterListViewModel {
 
     let filterTypeViewModels: [FilterTypeViewModel]
 
+    let shouldShowHistory: Bool
```

Comment:
Suggestion: IMO, setting the value as `false` here instead of the init will be it clear that product list doesn't support filters history. 
```suggestion
    let shouldShowHistory = false
```

Reply:
Hi there, I'd like to keep this as-is as I'm working on a PR to enable the filter history on the product list as well.